include/igl/boundary_loop.cpp

```diff
@@ -142,7 +142,7 @@ IGL_INLINE void igl::boundary_loop(
   vector<int> Lvec;
   boundary_loop(F,Lvec);
 
-  L.resize(Lvec.size());
```

Comment:
This will fail for RowVector (but that's not very popular so perhaps it's ok if it's a necessary change; not sure how it is).

Reply:
it dosent compile otherwise

Reply:
Probably doesn't compile because forwarding from a numpy-allocated array maybe the number of colums at compile time is not always known to be 1 (can be 0 and needs to be allocated). In any case, since the convention in libigl is to use column vectors, I would argue that passing a RowVector to a function returning a N x 1 matrix should be illegal anyway. No need to add an assert since the `resize()` will complain if the user does so.